### ISSUE TITLE
Flow matching objective

### ISSUE THREAD

**Issue by FloorVerhoeven** [VAST-AI-Research/DetailGen3D#2]
I was looking at your inference code and spotted that the variable name for the network's output suggests that the level of noise is being predicted:

https://github.com/VAST-AI-Research/DetailGen3D/blob/acdf05e1677c9aa7e656efddf4973e6b02ef1c24/detailgen3d/pipelines/pipeline_detailgen3d.py#L260

In the paper you state that you train with the flow matching objective for v-prediction. Given that the training code hasn't been made public, I cannot verify this myself. Can you please let me know whether you indeed train with a loss on the flow field velocity or if you're training to predict noise levels?

Also: are there any plans to release the training code?

**Comment by VLadImirluren** [comment]
Thanks for your attention and sorry to reply so late.
The noise_pred only represents the model_output, and we believe that the prediction of the flow model is v-prediction.
I have resigned from Vast, and there will be no update plan in the future. If you have any questions, feel free to contact me~
